src/runit.h

```diff
@@ -1,4 +1,4 @@
-#define RUNIT "/sbin/runit"
+#define RUNIT "/usr/bin/runit"
```

Comment:
I would argue that `/sbin/runit` is more correct. It can be used for all distributions pre-usr-merge.

Reply:
We do this change in the current `runit` template in void packages... It was added in b76d29be73683dd4318db7f336e66fbed5aef56e there.

Reply:
I'm gonna leave `/sbin/runit` and `/usr/bin/runit` open for discussion.